src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -118,6 +120,11 @@ public void deletePerson(ReadOnlyPerson target) throws PersonNotFoundException {
         }
 
         @Override
+        public void deleteTag(Tag target) throws TagNotFoundException {
```

Comment:
Should this deleteTag test be in AddCommandTest? Please practice Single Responsibility Principle